src/components/Contact/Contact.js

```diff
@@ -1,10 +1,23 @@
 import React, { Component } from 'react';
-import '../../styles/Contact.css';
+// import '../../styles/Contact.css';
+
+// import Header from './Header.js';
+// import Chat from './Chat.js';
+// import Help from './Help.js';
+// import SocialMediaIcons from './SocialMediaIcons.js'
+// import Follow from './Follow.js'
+// import Copyright from './Copyright.js'
 
 class Contact extends Component {
   render() {
     return (
       <div className="Contact" id="Contact">
+      // < Header />
+      // < Chat />
+      // < Help />
+      // < SocialMediaIcons />
+      // < Follow /> 
+      // < Copyright />
```

Comment:
Contact.js: Code is clean and well-organized for future additions/amendments. The next team member can remove `//`next to each line of code and use the template by Allan.